### PR TITLE
Updated nuki component configuration section

### PR DESCRIPTION
**Pull request by mnigbur**
Related to #6385.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/lock.nuki.markdown [hunk 1]
```diff
@@ -27,24 +27,21 @@ lock:
...
+{% configuration %}
+host:
+  description: The IP or hostname of the Nuki bridge.
+  required: true
+  type: string
+port:
+  description: The port on which the Nuki bridge is listening on.
+  required: false
+  default: 8080
+  type: int
```
Comment (frenck) [inline source/_components/lock.nuki.markdown]: `int` is not a valid type. Please use `integer`.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@mnigbur Thank you for your PR. I've left you a small comment. Could you please take a look? 👍 

Secondly, please open PR's to the current documentation to the `current` branch. This PR originally targeted the `next` branch, which is incorrect. I've updated this PR for you.

**Comment by mnigbur** [comment]
Uh, I'm sure i copied the port part from another component.. will hunt this down ;)

Thanks for fixing the PR target.. I'm still little confused with github.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for updating @mnigbur!

Don't worry about the git / GitHub confusing thing. We'll help you and you'll pick up on it in no time!
Keep doing this awesome work 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**